Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -175,6 +175,27 @@ TEST_CASE(test_png)
     TRY_OR_FAIL((test_roundtrip<Gfx::PNGWriter, Gfx::PNGImageDecoderPlugin>(TRY_OR_FAIL(create_test_rgba_bitmap()))));
 }
 
+TEST_CASE(test_png_paeth)
```

Comment:
Nit: you're testing the SIMD overload of the function, I think this should be explicited in the test name. Something like `test_png_paeth_simd`.

Reply:
Done.